app/src/main/java/com/example/euriskocodechallenge/common/utilityfunctions.kt

```diff
@@ -2,16 +2,69 @@ package com.example.euriskocodechallenge.common
 
 import android.app.Activity
 import android.content.Context
+import android.util.Base64
 import android.view.View
 import android.widget.Toast
 import com.example.euriskocodechallenge.utils.ConnectivityLiveData
 import java.security.AccessControlContext
+import javax.crypto.Cipher
+import javax.crypto.SecretKeyFactory
+import javax.crypto.spec.IvParameterSpec
+import javax.crypto.spec.PBEKeySpec
+import javax.crypto.spec.SecretKeySpec
 
 class utilityfunctions {
```

Comment:
@martin371998 class naming convention should start with UpperCase letter and follow CamelCasing

Reply:
Done in Utility functions branch